### PR TITLE
⚡ Remove unnecessary steps

### PR DESCRIPTION
**Pull request by david-martin**
* Remove node.js (Don't need this anymore)
* Remove clone of mobile-core repo (Don't need this anymore)
* Remove updating of master-config.yaml (Mobile extension & UI not developed this way anymore)
* Don't restart the origin pod (No config updates to apply anymore)

This may address intermittent start issues in #18. TBD

ping @psturc @philbrookes

### CODE REVIEW

File: mobile-core.addon [hunk 1]
```diff
@@ -3,41 +3,30 @@
...
-ssh bash /var/lib/minishift/openshift.local.config/install.sh
-ssh cd /var/lib/minishift/openshift.local.config/ && npm run grunt build
-docker restart origin
-ssh while [ $(yes | oc login -u system:admin | wc -l) -eq 0 ]; do sleep 2; done
-
-echo Installing Ansible Service Broker with Container Repository Organisation: #{CONTAINER_REPO_ORG}
+echo Installing Ansible Service Broker: repo=#{CORE_REPO},branch=#{CORE_BRANCH},container registry organisation=#{CONTAINER_REPO_ORG},container registry username=#{CONTAINER_REPO_USERNAME}
 echo
-ssh bash ~/mobile-core/installer/roles/ansible-service-broker-setup/files/provision-ansible-service-broker.sh '#{CONTAINER_REPO_USERNAME}' '#{CONTAINER_REPO_PASSWORD}' '#{CONTAINER_REPO_ORG}' true latest #{ip} nip.io ansible-service-broker
+ssh curl -s -O -J -L https://raw.githubusercontent.com/#{CORE_REPO}/mobile-core/#{CORE_BRANCH}~/installer/roles/ansible-service-broker-setup/files/provision-ansible-service-broker.sh
```
Comment (david-martin) [inline mobile-core.addon:27]: @philbrookes @maleck13  we may be able to simplify this a lot more (separate PR) if we leverage the broker addon https://github.com/minishift/minishift-addons/tree/master/add-ons/ansible-service-broker
Thoughts on doing that?

File: mobile-core.addon [hunk 2]
```diff
@@ -3,41 +3,30 @@
...
-
-ssh curl -s https://nodejs.org/dist/#{VER}/node-#{VER}-linux-x64.tar.gz > ~/#{VER}.tar.gz
-ssh cd /usr/local && sudo /usr/local/bin/tar --strip-components=1 -xzf ~/#{VER}.tar.gz
-
 ssh sudo chmod -R a+rwx /var/lib/minishift/openshift.local.config
 ssh curl -s -O -J -L https://github.com/openshift/origin/releases/download/v3.9.0/openshift-origin-client-tools-v3.9.0-191fece-linux-64bit.tar.gz
 ssh tar -xf openshift-origin-client-tools-v3.9.0-191fece-linux-64bit.tar.gz && sudo cp openshift-origin-client-tools-v3.9.0-191fece-linux-64bit/oc /usr/bin/oc
 ssh mkdir ~/.kube
 ssh cp /var/lib/minishift/openshift.local.config/master/admin.kubeconfig ~/.kube/config
 ssh oc login -u system:admin
```
Comment (david-martin) [inline mobile-core.addon:15]: This block is questionable if its required. I think its only required for the broker install below. See below comment for a suggestion for removing both this block and below broker install block

Comment (philbrookes) [reply mobile-core.addon:15]: This is currently only required for the waits now, I believe, like here:
https://github.com/aerogear/minishift-mobilecore-addon/pull/21/files#diff-00ef84954f4f9f6ae930aa96c1cf34b5R29

**Comment by philbrookes** [comment]
Verifying this locally

**Review by philbrookes — APPROVED** [review]

**Review by psturc — APPROVED** [review]
Verified

**Comment by david-martin** [comment]
Merging as is. It might fix/improve #18 (as we don't restart 'origin' anymore), but will not sure if it completely negates it yet.

**Pull request merged**